### ISSUE TITLE
508-defect-1: Missing lang attribute for in-page (non-global) Spanish PACT content

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10289]
## Describe the defect
On the PACT act page, when the user changes the page to the Spanish translation version there is no `lang` attribute indicating that the language has changed to es (Spanish). When the `lang` attribute is not set correctly, screen reader users hear the content pronounced incorrectly (with an English accent).

## To Reproduce
1. Go to The Spanish translation [PACT act page](https://www.va.gov/resources/the-pact-act-and-your-va-benefits-esp/)
2. Inspect the page and note that there is no lang attribute wrapping the main body content to denote the change in language
3. If available, turn on your screen reader and listen to the page being read and note that it is with an English accent

## AC / Expected behavior

- [ ] `lang=es` attribute has been added to indicate which section of the page content is in Spanish

## Screenshots
![image](https://user-images.githubusercontent.com/106678594/184934003-e60bc41a-c24d-42f5-8cb3-6f6774ec8bc9.png)

 ## Additional context
This issue is also occurring on the [COVID Vaccine page](https://www.va.gov/health-care/covid-19-vaccine-esp/) (both Spanish and Tagalog) so this is most likely an issue on all translation pages

I was speaking with Josh Kim and was told this was worked on and completed on that particular COVID page a while ago, I believe this is [the issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/22313) that relates to that work.  The issue is completed but the update is not longer on the production site, I confirmed with Josh he doesn't see it either.

## WCAG Guidance
[WCAG 2.0 A: Success Criterion 3.1.1: Language of Page](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html)

**Comment by laflannery** [comment]
For clarification:

On the PACT act page, there is an `<article>` tag. There should be a lang attribute on this element that switches between lang=en and lang=es depending on which language is toggled above

Current Screenshot:
![image](https://user-images.githubusercontent.com/106678594/184947242-f9d1062c-9e6b-47c4-8146-3a842530e797.png)

New English language screenshot: 
![image](https://user-images.githubusercontent.com/106678594/184947383-3d850fc5-3180-41a2-b771-fa98b3e4d253.png)

New Spanish Screenshot:
![image](https://user-images.githubusercontent.com/106678594/184948060-6612160d-1ba1-4909-8a72-743b4fdc269b.png)

The COVID page does have this attribute - it was missed before because it had been moved from the original location.  I believe we were mistaken and this is only applying to the new PACT act page

**Comment by laflannery** [comment]
For the COVID page - it looks like this is a larger screen reader bug that we cannot do anything about which is why it was not behaving as expected.  I am going to update the description to indicate that this is not a sitewide issue

**Comment by allisonlu** [comment]
Looks like the `article` element is needs the class `usa-content` so that `setLangAttributes` function can execute. See [line 104 in this file on the **`vets-website`** repo](https://github.com/department-of-veterans-affairs/vets-website/blob/4a4e62e7c6f56c4ff90da2b01b1b238f75350d94/src/applications/static-pages/i18Select/utilities/helpers.js#L104). 

I still need to pull down this new content onto my local, which may take a while so I'll test this change tomorrow!

**Comment by wesrowe** [comment]
@laflannery, Dave C asked me to push a little on your statement that the COVID page has a different issue. It's very important to us to make sure the templates do the right thing automatically, and that any manual changes required for translation are clearly captured in our "how to translation" documentation/playbook. (Which might at this point be just findings in a discovery ticket.)

So @allisonlu, 
1. Please take the time to understand the nature of the issue on the COVID page also.
2. Also, can you create updated documentation about "how to translation"? I.e., adapt the original discovery ticket findings into a more findable place on github. (Bump up estimate if necessary.)

Cc @jilladams @davidconlon

**Comment by wesrowe** [comment]
@laflannery, is there a work-around that we can implement? Like if a common screen reader is looking for the `lang` attribute elsewhere in the dom, I would consider adding it there also.

**Comment by laflannery** [comment]
@wesrowe Unfortunately I do not know of one and I wasn't able to find one using a simple test page I created

**Comment by allisonlu** [comment]
PR opened on `content-build`: https://github.com/department-of-veterans-affairs/content-build/pull/1268

**Comment by allisonlu** [comment]
PR merged and fix will be going out with Monday's deploy!

**Comment by allisonlu** [comment]
This fix is now on prod!

https://www.va.gov/resources/the-pact-act-and-your-va-benefits-esp/

https://www.va.gov/resources/the-pact-act-and-your-va-benefits-tag/

**Comment by laflannery** [comment]
This looks great, thank you!
